### DISCUSSION TITLE
Where is the option to download the workflow configuration to a yaml file?

### DISCUSSION THREAD

**Discussion by WilkoThomassen** [codemagic-ci-cd/codemagic-docs#1549] [Q&A] [answered]
I was using the functionality to export the workflow configuration a lot. According to the [Codemagic Cheat sheet](https://blog.codemagic.io/codemagic-yaml-cheatsheet/) it should still be there. Is this functionality completely removed? 

Is it still possible to export my workflow to yaml? how?

**Comment by himesh-cm** [comment, marked as answer]
Hi @WilkoThomassen, this option was removed. You can refer to our sample projects [here](https://github.com/codemagic-ci-cd/codemagic-sample-projects/tree/main/flutter/flutter-android-and-ios-yaml-demo-project), which should work with minimal changes. Let me know if you need assistance with the yaml workflow.
